R/Dataset.R

```diff
@@ -9,6 +9,9 @@ Dataset <- function(data, target, type = c("regression", "classification"),
   if(!target %in% colnames(data)) {
```

Comment:
I belive type should be an optional argument as per the instructions?
Maybe we can also figure this out by looking at the target columns distinct values?
e.g. More than 10 distinct values -> Regression
or type == int or factor -> Classification